src/en/guides/ict-requirements.html

```diff
@@ -4,81 +4,42 @@
-		members and non-members, and can be found in ETSI SR 000 314: "Intellectual Property Rights (IPRs); Essential,
-		or potentially Essential, IPRs notified to ETSI in respect of ETSI standards", which is available from the ETSI
-		Secretariat. Latest updates are available on the ETSI Web server <a
-			href="https://ipr.etsi.org/">(https://ipr.etsi.org/)</a>. Please visit <a href="https://ipr.etsi.org/">ETSI
-			IPR database</a> to search for any IPRs.</p>
-	<p>At first glance, some requirements may appear to be unrelated to this product or service. They have been included
-		for consideration since the full feature set of a Vendor’s product or service may not be known. For example, a
-		video may be embedded into product documentation, so accessibility requirements for video and audio may become
-		relevant.</p>
+	<p>This document reproduces relevant ICT accessibility requirements from the EN 301 549 v3.2.1 (2021-03) Harmonised European Standard – Accessibility requirements for ICT products and services (© used under license from European Telecommunications Standards Institute 2021. © Comité Européen de Normalisation 2021. © Comité Européen de Normalisation Électrotechnique 2021. All rights reserved.), which includes the Web Content Accessibility Guidelines (WCAG) 2.1 level AA. Note, where the EN 301 549 references VOID criteria, for the purposes of this document, we have excluded the technical explanation as to why the WCAG criteria no longer apply. Refer to the EN 301 549 (hyperlink below) for this information if necessary.</p>
+	<p>Intellectual Property Rights (IPRs) essential or potentially essential to normative deliverables may have been declared to ETSI. The information pertaining to these essential IPRs, if any, is publicly available for ETSI members and non-members, and can be found in ETSI SR 000 314: "Intellectual Property Rights (IPRs); Essential, or potentially Essential, IPRs notified to ETSI in respect of ETSI standards", which is available from the ETSI Secretariat. Latest updates are available on the ETSI Web server <a href="https://ipr.etsi.org/">(https://ipr.etsi.org/)</a>. Please visit <a href="https://ipr.etsi.org/">ETSI IPR database</a> to search for any IPRs.</p>
+	<p>At first glance, some requirements may appear to be unrelated to this product or service. They have been included for consideration since the full feature set of a Vendor’s product or service may not be known. For example, a video may be embedded into product documentation, so accessibility requirements for video and audio may become relevant.</p>
 	<p>Appendices include definitions, references, and practical guidance on creating accessible documentation.</p>
 	<p>Internal to this document references are included but not always linked (footnotes or otherwise).</p>
-</div>
-<strong>Sources used to compile this document</strong>
+</div> <strong>Sources used to compile this document</strong>
```

Comment:
Can you wrap this `<strong>` in a `<p>`? Same in the French version.

Reply:
oh sure

Reply:
I think there is a problem here, in the French version the "<strong>Sources used to compile this document</strong>" is written as "<h2>Sources utilisées pour compiler le présent document</h2>"

Reply:
i changed the french version to look like the english version as that make most sense, let me know if it is right so i can push it, otherwise you can suggest any better or correct way